scriptworker/log.py

```diff
@@ -88,7 +88,8 @@ def get_log_filenames(context):
     Returns:
         tuple: log file path, error log file path
     """
-    log_file = os.path.join(context.config['task_log_dir'], 'task_output.log')
+    # XXX Even though our logs aren't live, Treeherder looks for live_backing.log to show errors in failures summary
+    log_file = os.path.join(context.config['task_log_dir'], 'live_backing.log')
```

Comment:
I agree. In practice, I haven't noticed a the difference between the 2 files (except task_output.log contains the exit code at the end). I'll get rid task_output in this PR.